### PR TITLE
[Ne] Translated to Nepali Language (Validation-inline)

### PR DESCRIPTION
**Pull request by sushantpoudel98**
Every Words are accurate and native. Being a fellow Nepali, I can assure he is an expert in Nepali typing, writing and reading.

### CODE REVIEW

File: src/ne/auth.php [hunk 1]
```diff
@@ -13,5 +13,5 @@
     */
 
     'failed'   => 'यी प्रमाणहरू हाम्रो रेकर्ड संग मेल खादैनन्।',
-    'throttle' => 'धेरै लगिन प्रयास भयो। :seconds सेकेन्ड पछि फेरि प्रयास गर्नुहोस्।',
+    'throttle' => 'धेरै लगिन प्रयास भयो। :सेकेन्ड सेकेन्ड पछि फेरि प्रयास गर्नुहोस्।',
```
Comment (caouecs) [inline src/ne/auth.php:16]: Don't translate tag

File: src/ne/nepali.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?php
```
Comment (caouecs) [inline src/ne/nepali.php:1]: Just translate official files

File: src/ne/pagination.php [hunk 3]
```diff
@@ -12,6 +12,6 @@
     |
     */
 
-    'previous' => '&laquo; अघिल्लो',
-    'next'     => 'अर्को &raquo;',
+    'previous' => 'पछाडी; अधिकाे',
```
Comment (caouecs) [inline src/ne/pagination.php:15]: don't translate html

File: src/ne/pagination.php [hunk 4]
```diff
@@ -12,6 +12,6 @@
...
+    'next' => 'अर्काे; अगाडी;',
```
Comment (caouecs) [inline src/ne/pagination.php:16]: don't translate html and don't remove space

File: src/ne/passwords.php [hunk 5]
```diff
@@ -12,9 +12,25 @@
...
 
+    'Laravel'   => 'लाराभेल',
     'reset'     => 'पासवर्ड रिसेट भयो!',
+    'Dashboard' => 'ड्यासबोर्ड',
     'sent'      => 'हामीले पासवर्ड परिवर्तन गर्ने लिंक ईमेलमा पठाएका छौ!',
-    'throttled' => 'Please wait before retrying.',
+    'throttled' => 'पुन: प्रयास गर्नु अघि कृपया प्रतीक्षा गर्नुहोस्.',
     'token'     => 'यो पासवर्ड रिसेट गर्ने टोकन अमान्य छ।',
     'user'      => 'दिइएको ईमेल प्रयोगकर्ता भेटिएन।',
+    'Password'  => 'पासवर्ड',
```
Comment (caouecs) [inline src/ne/passwords.php:22]: don't add not official tags

File: src/ne/validation-inline.php [hunk 6]
```diff
@@ -13,110 +13,110 @@
...
-    'before_or_equal' => 'This must be a date before or equal to :date.',
+    'accepted'        => 'यो क्षेत्र स्वीकार गर्नु पर्छ।',
+    'active_url'      => 'यो मान्य URL होईन।',
+    'after'           => 'यो मिति पछाडि मिति हुनुपर्दछ।',
+    'after_or_equal'  => 'यो मिति पछाडि वा बराबर मिति हुनुपर्दछ।',
+    'alpha'           => 'यो फिल्डमा अक्षरहरू मात्र समावेश हुन सक्छ।',
+    'alpha_dash'      => 'यस फाँटमा केवल अक्षरहरू, संख्याहरू, ड्यासहरू र अन्डरस्कोरहरू समावेश हुन सक्छ।',
+    'alpha_num'       => 'यो फिल्डमा अक्षर र संख्याहरू मात्र समावेश हुन सक्छ।',
+    'array'           => 'यो फिल्ड एरे हुनै पर्छ।',
+    'before'          => 'यो मिति भन्दा अघि मिति हुनुपर्दछ।',
```
Comment (caouecs) [inline src/ne/validation-inline.php:24]: don't translate tags

**Comment by dbasechhetri** [comment]
Ok ! got it . Thank You for valuable suggestion. I will try to fellow format and syntax.

File: src/ne/auth.php [hunk 7]
```diff
@@ -13,5 +13,5 @@
     */
 
     'failed'   => 'यी प्रमाणहरू हाम्रो रेकर्ड संग मेल खादैनन्।',
-    'throttle' => 'धेरै लगिन प्रयास भयो। :seconds सेकेन्ड पछि फेरि प्रयास गर्नुहोस्।',
+    'throttle' => 'धेरै लगिन प्रयास भयो। :second सेकेन्ड पछि फेरि प्रयास गर्नुहोस्।',
```
Comment (caouecs) [inline src/ne/auth.php:16]: :seconds

File: src/ne/passwords.php [hunk 8]
```diff
@@ -14,7 +14,7 @@
 
     'reset'     => 'पासवर्ड रिसेट भयो!',
     'sent'      => 'हामीले पासवर्ड परिवर्तन गर्ने लिंक ईमेलमा पठाएका छौ!',
-    'throttled' => 'Please wait before retrying.',
+    'throttled' => 'पुन: प्रयास गर्नु अघि कृपया प्रतीक्षा गर्नुहोस्.',
     'token'     => 'यो पासवर्ड रिसेट गर्ने टोकन अमान्य छ।',
     'user'      => 'दिइएको ईमेल प्रयोगकर्ता भेटिएन।',
-];
+    ];
```
Comment (caouecs) [inline src/ne/passwords.php:20]: No space

File: src/ne/validation.php [hunk 9]
```diff
@@ -15,106 +15,106 @@
...
-    'between'         => [
-        'numeric' => ':attribute :min र :max को बिचमा हुनुपर्छ।',
-        'file'    => ':attribute :min र :max kilobytes को बिचमा हुनुपर्छ।',
+    'before_or_equal' => ':attribute मिति भन्दा अघि वा बराबर मिति हुनुपर्दछ। ',
+    'between'=> [
+        'numeric' => ':attribute :min र :maxको बिचमा हुनुपर्छ।',
+        'file'    => ':attribute :min र :max किलोबाइट्स को बिचमा हुनुपर्छ।',
         'string'  => ':attribute :min र :max वर्णको बिचमा हुनुपर्छ।',
-        'array'   => ':attribute मा आइटमको संख्या :min र :max को बिचमा हुनुपर्छ।',
+        'array'   => ':attribute :आइटमको संख्या :min र :max को बिचमा हुनुपर्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:29]: Why ":" ?

File: src/ne/validation.php [hunk 10]
```diff
@@ -15,106 +15,106 @@
...
     ],
-    'boolean'        => ':attribute ठिक अथवा बेठिक हुनुपर्छ।',
+    'boolean'        => ':attribute :true अथवा :false हुनुपर्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:31]: Why :true and :false ?

File: src/ne/validation.php [hunk 11]
```diff
@@ -15,106 +15,106 @@
...
     'confirmed'      => ':attribute दाेहाेर्याइएकाे मिलेन।',
     'date'           => ':attribute को मिति मिलेन।',
-    'date_equals'    => 'The :attribute must be a date equal to :date.',
+    'date_equals'    => ':value मिति बराबर मिति हुनुपर्दछ।',
```
Comment (caouecs) [inline src/ne/validation.php:34]: :attribute and :date

File: src/ne/validation.php [hunk 12]
```diff
@@ -15,106 +15,106 @@
...
     'date_format'    => ':attribute को ढाँचा :format जस्तो हुनुपर्छ।',
-    'different'      => ':attribute र :other फरक हुनुपर्छ।',
+    'different'      => ':attribute र :फरक फरक हुनुपर्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:36]: :other

File: src/ne/validation.php [hunk 13]
```diff
@@ -15,106 +15,106 @@
...
     'digits'         => ':attribute :digits अंकको हुनुपर्छ।',
-    'digits_between' => ':attribute :min देखि :max अंकको हुनुपर्छ।',
-    'dimensions'     => 'The :attribute has invalid image dimensions.',
-    'distinct'       => 'The :attribute field has a duplicate value.',
+    'digits_between' => ':attribute :min देखी :max अंकको हुनुपर्छ।',
+    'dimensions'     => ':attribute अमान्य छवि आयाम छ।',
+    'distinct'       => ':attribute फिल्ड फिल्डको नक्कल मान छ',
     'email'          => ':attribute को इमेल ठेगाना मिलेन।',
-    'ends_with'      => 'The :attribute must end with one of the following: :values.',
+    'ends_with'      => ':attribute निम्न मध्ये एकको साथ विशेषता अन्त हुनुपर्दछ :value',
```
Comment (caouecs) [inline src/ne/validation.php:42]: :values

File: src/ne/validation.php [hunk 14]
```diff
@@ -15,106 +15,106 @@
...
     ],
     'max' => [
-        'numeric' => ':attribute :max भन्दा बढी हुनुहुदैन।',
-        'file'    => ':attribute :max kilobytes भन्दा बढी हुनुहुदैन।',
-        'string'  => ':attribute :max वर्ण भन्दा बढी हुनुहुदैन।',
-        'array'   => ':attribute मा :max भन्दा बढी आइटम हुनुहुदैन।',
+        'numeric' => ' :attribute मान भन्दा :max हुनुपर्दछ।',
+        'file'    => ' :attribute मान क्यालोबाइट भन्दा :max हुनुपर्दछ।',
+        'string'  => ' :attribute मान क्यारेक्टर भन्दा :max हुनुपर्दछ।',
+        'array'   => ' :attribute मान आईटमहरू भन्दा बढि हुनुपर्दछ।',
```
Comment (caouecs) [inline src/ne/validation.php:82]: :max

File: src/ne/validation.php [hunk 15]
```diff
@@ -15,106 +15,106 @@
...
+    'present'              => ':attribute क्षेत्र उपस्थित हुनुपर्दछ.',
     'regex'                => ':attribute को ढाँचा मिलेन।',
     'required'             => ':attribute दिइएको हुनुपर्छ।',
     'required_if'          => ':attribute चाहिन्छ जब :other :value हुन्छ।',
-    'required_unless'      => ':other :values मा नभएसम्म :attribute चाहिन्छ।',
-    'required_with'        => ':values भएसम्म :attribute चाहिन्छ।',
-    'required_with_all'    => ':values भएसम्म :attribute चाहिन्छ।',
-    'required_without'     => ':values नभएको बेला :attribute चाहिन्छ।',
-    'required_without_all' => 'कुनैपनि :values नभएको बेला :attribute चाहिन्छ।',
+    'required_unless'      => ':other :value मा नभएसम्म :attribute चाहिन्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:99]: :values

File: src/ne/validation.php [hunk 16]
```diff
@@ -15,106 +15,106 @@
...
+    'required_with'        => ':value भएसम्म :attribute चाहिन्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:100]: :values

File: src/ne/validation.php [hunk 17]
```diff
@@ -15,106 +15,106 @@
...
+    'required_with_all'    => ':value भएसम्म :attribute चाहिन्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:101]: :values

File: src/ne/validation.php [hunk 18]
```diff
@@ -15,106 +15,106 @@
...
+    'required_without'     => ':value नभएको बेला :attribute चाहिन्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:102]: :values

File: src/ne/validation.php [hunk 19]
```diff
@@ -15,106 +15,106 @@
...
+    'required_without_all' => 'कुनैपनि :value नभएको बेला :attribute चाहिन्छ।',
```
Comment (caouecs) [inline src/ne/validation.php:103]: :values

File: src/ne/validation.php [hunk 20]
```diff
@@ -15,106 +15,106 @@
...
         'numeric' => ':attribute :size हुनुपर्छ।',
-        'file'    => ':attribute :size kilobytesको हुनुपर्छ।',
+        'file'    => ':attribute :size क्यालोबाइटोकाे हुनुपर्छ।',
         'string'  => ':attribute :size वर्णको हुनुपर्छ।.',
-        'array'   => ':attribute मा :size आइटम हुनुपर्छ।',
+        'array'   => ':attribute :sizeमा आइटम हुनुपर्छ।',
     ],
-    'starts_with' => 'The :attribute must start with one of the following: :values.',
-    'string'      => ':attribute string हुनुपर्छ।',
+    'starts_with' => ':attribute निम्न मध्ये एकसँग सुरू हुनुपर्दछ :others',
```
Comment (caouecs) [inline src/ne/validation.php:111]: :values not :others

**Comment by dbasechhetri** [comment]
Thank You for assisting.

File: src/ne/validation.php [hunk 21]
```diff
@@ -15,84 +15,84 @@
...
     'alpha_dash'      => ':attribute मा अक्षर, संख्या र ड्यासहरू मात्र हुनसक्छ।',
     'alpha_num'       => ':attribute मा अक्षर र संख्याहरू मात्र हुनसक्छ।',
-    'array'           => ':attribute array हुनुपर्छ।',
+    'array'           => ':attribute एर्रे हुनुपर्छ।',
     'before'          => ':attribute को मिति :date भन्दा अघि हुनुपर्छ।',
-    'before_or_equal' => 'The :attribute must be a date before or equal to :date.',
-    'between'         => [
-        'numeric' => ':attribute :min र :max को बिचमा हुनुपर्छ।',
-        'file'    => ':attribute :min र :max kilobytes को बिचमा हुनुपर्छ।',
+    'before_or_equal' => ':attribute मिति भन्दा अघि वा बराबर मिति हुनुपर्दछ। ',
```
Comment (caouecs) [inline src/ne/validation.php:24]: :date

File: src/ne/validation.php [hunk 22]
```diff
@@ -15,84 +15,84 @@
...
+    'between' => [
```
Comment (caouecs) [inline src/ne/validation.php:25]: space

**Comment by dbasechhetri** [comment]
All Done !

**Comment by caouecs** [comment]
Thank you

**Pull request merged**